src/components/Dropdown/ChainType.tsx

```diff
@@ -73,27 +72,27 @@ export default ({ setShowChainDropdown, left }: { setShowChainDropdown: Function
       }}
     >
       <div
-        className={`chain_type_${chainType === 'main' ? 'selected' : 'normal'}`}
+        className={`chain_type_${isMainnet() ? 'selected' : 'normal'}`}
         role="button"
         tabIndex={-1}
         onKeyDown={() => {}}
         onClick={() => {
           setShowChainDropdown(false)
         }}
       >
-        {chainType === 'main' ? i18n.t('blockchain.mainnet') : i18n.t('blockchain.mainnet_coming')}
+        <a href={CONFIG.MAINNET_URL}>{i18n.t('blockchain.mainnet')}</a>
       </div>
       <div className="chain_type_separate" />
       <div
-        className={`chain_type_${chainType !== 'main' ? 'selected' : 'normal'}`}
+        className={`chain_type_${!isMainnet() ? 'selected' : 'normal'}`}
         role="button"
         tabIndex={-1}
         onKeyDown={() => {}}
         onClick={() => {
           setShowChainDropdown(false)
         }}
       >
-        {chainType === 'ckb_test' ? i18n.t('blockchain.testnet') : i18n.t('blockchain.testnet')}
+        <a href={`${CONFIG.MAINNET_URL}/${CONFIG.TESTNET_NAME}`}>{i18n.t('blockchain.testnet')}</a>
```

Comment:
why use `mainnet_url` connect `testnet_name`?

Reply:
`testnet_url` = `mainnet_url` + / + `testname`

Reply:
maybe using constant `testnet_url` replace `${CONFIG.MAINNET_URL}/${CONFIG.TESTNET_NAME}` would be better. `testnet_url = mainnet_url + / + testnet_name`